Formula/a/act_runner.rb

```diff
@@ -0,0 +1,39 @@
+class ActRunner < Formula
+  desc "Action runner for Gitea based on Gitea's fork of act"
+  homepage "https://docs.gitea.com/usage/actions/act-runner"
+  url "https://gitea.com/gitea/act_runner/archive/v0.2.11.tar.gz"
+  sha256 "8b317700d1f3344d8664be9edb004914723a4aacc8f8b1b3719ca2260a5866b6"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    # Upstream uses a Makefile for builds instead of `go build`.
+    #
+    # The `-s -w` LDFLAGS are already provided in the Makefile, and go's linker
+    # doesn't process the `-L/opt/homebrew/opt/ruby/lib` flag correctly (it
+    # complains about the lack of space between the flag and the path), so
+    # LDFLAGS is set to empty here.
+    #
+    # GOFLAGS is used to pass the `-trimpath` flag in `std_go_args` since the
+    # Makefile hardcodes the `go build` invocation. Unfortunately, according to
+    # the `go` command's documentation on environment variables, they get
+    # overridden by the command line flags, so the `-o` flag can't be passed
+    # this way and the binary will need to be copied manually:
+    # https://pkg.go.dev/cmd/go#hdr-Environment_variables
+    system "make", "build", "LDFLAGS=", "GOFLAGS=-trimpath"
+    bin.install "act_runner"
+  end
+
+  test do
+    # This test verifies that the command fails (with exit code 1) to register
+    # with an invalid token and that it prints the failure to register to
+    # stderr.
+    assert_match "Error: Failed to register runner",
+                 shell_output("#{bin}/act_runner register " \
+                              "--no-interactive " \
+                              "--instance https://gitea.com " \
+                              "--token INVALID_TOKEN " \
+                              "2>&1", 1)
```

Comment:
I think comment can be removed.
```suggestion
    args = %w[
      --no-interactive
      --instance https://gitea.com
      --token INVALID_TOKEN
    ]
    output = shell_output("#{bin}/act_runner register #{args.join(" ")} 2>&1", 1)
    assert_match "Error: Failed to register runner", output
```